projects/ngx-translate-cache/src/lib/ngx-translate-cache.service.ts

```diff
@@ -10,12 +10,15 @@ export namespace CacheMechanism {
 export const CACHE_NAME = new InjectionToken<string>('CACHE_NAME');
 export const CACHE_MECHANISM = new InjectionToken<string>('CACHE_MECHANISM');
 export const COOKIE_EXPIRY = new InjectionToken<string>('COOKIE_EXPIRY');
+export const COOKIE_ATTRIBUTES = new InjectionToken<string>('COOKIE_ATTRIBUTES');
+export const COOKIE_EXPIRE_SESSION = -1;
```

Comment:
Could we consider interpreting any negative value as a session cookie?

Reply:
That would be a fair interpretation. See most recent commit.